Sockets/BackEndMonitoringSockets/src/CAcceptorWrapper.cpp

```diff
@@ -2,38 +2,40 @@
 #include "CAcceptorWrapper.h"
 
 CAcceptorWrapper::CAcceptorWrapper(int port, const std::string& ip_address, 
-	size_t num_threads, CEvent& event, std::shared_ptr<CLogger> logger)
-	: m_event(event), m_logger(logger)
+	CEvent& event, std::shared_ptr<CThreadPool> pool, 
+	std::shared_ptr<CLogger> logger)
+	: m_event(event), m_pool(pool), m_logger(logger)
 {
 	m_server_acceptor = InitAcceptor(1111, "127.0.0.1");
 	m_stream = InitSocketWrapper(m_server_acceptor->GetHandle());
 	m_service_handler = InitServiceHandler(m_server_acceptor->GetHandle());
```

Comment:
The constructor shouldn't implement hard init. Better to add another method for such initialization and added variable that check is_initialized object

Reply:
@Serwlashev, and this